bulk-model-sync-gradle/src/main/kotlin/org/modelix/model/sync/bulk/gradle/config/ModelSyncGradleSettings.kt

```diff
@@ -95,6 +95,25 @@ sealed interface LocalEndpoint : SyncEndpoint {
     var repositoryDir: File?
     var mpsDebugPort: Int?
 
+    /**
+     * Add a plugin to be loaded when running MPS.
+     * `jetbrains.mps.core`, `jetbrains.mps.testing`, `jetbrains.mps.ide.make` are always loaded.
+     *
+     * All other plugins, even bundled ones, must be configured explicitly.
+     * In general cases, the sync does not rely on concepts (and in turn on languages and plugins) of the synced nodes.
+     *
+     * Loading other plugins might become necessary when they provide custom persistence
+     * and in other, yet unknown cases.
+     * First, try if the sync works for your project without adding plugins.
+     *
+     * Example usage:
+     * ```
+     * mpsPlugin(BundledPluginSpec("jetbrains.mps.vcs", File("plugins/mps-vcs")))
```

Comment:
The path for bundled plugins cannot be resolved automatically?

Reply:
It would be possible to only specify the ID and search for the folder the contains the `META-INF/plugin.xml`with that ID.

But I wanted to keep it simple and explicit.
Also I am not sure if Plugins always must have a META-INF/plugin.xml